packages/semantic-ui/src/components/List.js

```diff
@@ -331,6 +351,36 @@ const useList = (WrappedComponent: ComponentType<any>) => (
       this.setState({ selectedItem: copy, modalEdit: true });
     }
 
+    /**
+     * Generates and downloads a CSV file containing all
+     * the data in the table.
+     *
+     * @param items
+     */
+    onCsvExportButton() {
+      let csv = `${this.props.columns.map((col) => `"${col.label}"`).join(',')}\n`;
```

Comment:
Should we only include visible columns to allow a user to export a subset of all available columns?